package/gluon-l3roamd/luasrc/lib/gluon/upgrade/380-gluon-l3roamd-route

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/lua
+
+local uci = require('simple-uci').cursor()
+local site = require 'gluon.site_config'
+
+uci:delete('network', 'l3roam')
+uci:section('network', 'interface', 'l3roam',
+{
+	ifname = 'l3roam0',
+	proto = 'none',
+}
+)
```

Comment:
Please adjust the coding style to match our other scripts: opening `{` on the `uci:section` line, closing `})` on a single line.